README.md

```diff
@@ -233,3 +233,16 @@ Split a cell horizontally, where `t` is a value ranged 0 to 1 representing where
 ### updateNormals(quads, cell)
 
 Updates all of the normals of a cell to be that of the computed normal of the cell.
+
+### fromFunction(f, nu, nv, computeNormals = true, uPeriodic = false, vPeriodic = false)
```

Comment:
I'm starting to think this module shouldn't compute normals by looking at neighbors. If you're already giving it a continuous function, it seems much more reasonable to just use numerical differentiation Long story short, I think these flags need a major overhaul.